manifests/deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: weather-web
+  annotations:
+    keel.sh/policy: all
+    keel.sh/trigger: poll
+  name: weather-web
+  namespace: oz-lab
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: weather-web
+  strategy:
+    rollingUpdate:
+      maxUnavailable: 50%
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: weather-web
+    spec:
+      containers:
+      - image: ghcr.io/nataliewolfe/weather-web:v0.1.0
```

Comment:
Why is this pinned to v0.1.0? Will I have to update this file every release?

Reply:
No, the CD (Keel) will update the manifest on the fly. In the event that the manifest needs to be re-applied from configuration, this tag should be updated if the upstream tag has changed. This should only happen if the manifest is changing for another reason.

I'll work through a better solution with how the kubernetes code is tracked in this code base that doesn't rely on hand-coding tag values.